recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: wntr
+  version: "0.2.0"
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/3c8a669d50e77e872e9e6dd2c391a9fa83c34ecd.zip
+  sha256: 5d33a18f7cee0f76ca7a99074243a5019ae70f76ccb84dc44be54106eb758589
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+    - pip
+  host:
+    - python
+    - pip
```

Comment:
please, remove ``pip`` from here
I just saw that this project depends of anothers lobraries such as:
networkx, pandas, numpy, scipy, matplotlib, plotly

So, you have to add them to the host requirements and also to the run requirements

Reply:
@marcelotrevisani Ah, thanks! Sorry - I am new to this. I will fix it now.

Reply:
No problem! We are here to learn  :)